app/models/language_exporter.rb

```diff
@@ -442,12 +442,12 @@ def validate_square_brackets(value)
     value = value.to_s.dup
     pass = true
     while value =~ /\S/
-      if value.sub!(/^[^\[\]]+/, "")
-      elsif value.sub!(/^\[\[/, "")
-      elsif value.sub!(/^\]\]/, "")
-      elsif value.sub!(/^\[\w+\]/, "")
-      elsif value.sub!(/^\[:\w+(?:\(([^\[\]]+)\))?\]/, "")
-        if Regexp.last_match(1) && \
+      if value.sub!(/^[^\[\]]+/, "") ||
+         value.sub!(/^\[\[/, "") ||
+         value.sub!(/^\]\]/, "") ||
+         value.sub!(/^\[\w+\]/, "") ||
+         value.sub!(/^\[:\w+(?:\(([^\[\]]+)\))?\]/, "")
```

Comment:
Fixes Lint/EmptyConditionalBody offenses

Reply:
Hmmm, this doesn't look the same to me.  How about:

while value =~ /\S/
      next if value.sub!(/^[^\[\]]+/, "") ||
              value.sub!(/^\[\[/, "") ||
              value.sub!(/^\]\]/, "") ||
              value.sub!(/^\[\w+\]/, "")

      if value.sub!(/^\[:\w+(?:\(([^\[\]]+)\))?\]/, "")
        if Regexp.last_match(1) && \
           !validate_square_brackets_args(Regexp.last_match(1))
          pass = false
          break
        end
      else
        pass = false
        break
      end
    end

Reply:
Or maybe this is cleaner?

    while value =~ /\S/
      next if value.sub!(/^[^\[\]]+/, "") ||
              value.sub!(/^\[\[/, "") ||
              value.sub!(/^\]\]/, "") ||
              value.sub!(/^\[\w+\]/, "") ||
              value.sub!(/^\[:\w+(?:\(([^\[\]]+)\))?\]/, "") &&
                (!Regexp.last_match(1) ||
                validate_square_brackets_args(Regexp.last_match(1)))
      pass = false
      break
    end

Reply:
I find this whole thing frustrating, but I have to move on.  What it comes down to is it's well tested.  If the tests pass, it must be logically equivalent.

Reply:
Thanks for the additional feedback. I don't want to frustrate you, so I'll look at it again when I'm more awake.

Reply:
It's rubocop that's frustrating me, not you.  It's fine most of the time, even occasionally helpful, but when it breaks things, makes things uglier or wastes time it is frustrating.